logging.rst

```diff
@@ -92,8 +92,14 @@ In the ``prod`` environment, logs are written to `STDERR PHP stream`_, which
 works best in modern containerized applications deployed to servers without
 disk write permissions.
 
-If you prefer to store production logs in a file, set the ``path`` of your
-log handler(s) to the path of the file to use (e.g. ``var/log/prod.log``).
+If you prefer to store production logs in a file, set the ``path`` option of your
+log handler(s) to the desired file path (e.g. ``var/log/prod.log``). This directory
+is created by Monolog if needed, so Symfony applications running on read-only
```

Comment:
If it is a read only filesystem, how can it be created?

Reply:
Agreed, If I understand, this is useful when symfony app is readonly and logs folder is defined by`APP_LOG_DIR` outside symfony app, for example `/tmp/log`. In this case symfony app keep readonly